src/frontend/apps/impress/src/features/docs/doc-export/components/ModalExport.tsx

```diff
@@ -92,10 +92,15 @@ export const ModalExport = ({ onClose, doc }: ModalExportProps) => {
       const exporter = new PDFExporter(editor.schema, pdfDocsSchemaMappings, {
         resolveFileUrl: async (url) => exportCorsResolveFileUrl(doc.id, url),
       });
-      const pdfDocument = (await exporter.toReactPDFDocument(
+      const rawPdfDocument = (await exporter.toReactPDFDocument(
         exportDocument,
       )) as React.ReactElement<DocumentProps>;
 
+      // Inject language="fr-FR" for screen reader support
+      const pdfDocument = isValidElement(rawPdfDocument)
+        ? cloneElement(rawPdfDocument, { language: 'fr-FR' })
```

Comment:
The site is not everytime in french, it can be in any language actually, you must use `i18n.language` I think.